src/index.coffee

```diff
@@ -23,10 +23,11 @@ module.exports = class AutoLaunch
         versions = process?.versions
         if path?
             # Verify that the path is absolute
-            throw new Error 'path must be absolute' unless isPathAbsolute path
+            throw new Error 'path must be absolute' unless pathTools.isAbsolute path

```

Comment:
Looks like `path.isAbsolute` was introduced into node.js in version 11. I think we have support version 8 and up.

Best to leave it as is on this pr unless it's necessary. Be nice to drop support for really old versions of node but but as a specific change and major release.